README.md

```diff
@@ -1,26 +1,96 @@
+template ourselves.
 
 ## Quickstart
 
-- If you don't already have it, install [pixi](https://pixi.sh/latest/#installation)
-- Clone this repository
+### 1) Install Copier
+
+This template is based on [Copier](https://copier.readthedocs.io), a tool to
+copy files and directories from a template repository to a destination
+directory, while allowing you to answer questions to customize the
+destination. You can install `copier` in several ways, depending on your
+environment and preferences. Here are some common methods, please choose one
+that suits you best:
+
+
+#### Homebrew (OSX)
+
+If you are on OSX and have [Homebrew](https://brew.sh/), you can install
+`copier` globally with the following command:
+
+```bash
+brew install copier
+```
+
+#### Pixi (contributors)
+
+Install [Pixi](https://pixi.sh/latest/#installation) and:
+
+- Clone this repository and activate it's environment:
   ```bash
   git clone https://github.com/castelao/NREL-pypackage-template.git
   cd NREL-pypackage-template
+  pixi shell
   ```
-- Create your new project by running
-  ```bash
-  pixi run copier copy https://github.com/castelao/NREL-pypackage-template ../my_new_project
-  cd ../my_new_project
-  ```
-  You should see several files and directories created from the information
-  that you gave. For instance, take a look at the `pyproject.toml` file.
-- Your package will probably depend on other libraries. Let's assume that
-  you'll use `xarray`, so you can run
+
+- Or create your own environment:
   ```bash
-  pixi add xarray
+  pixi init my_project_creator
+  pixi add copier
+  pixi shell
   ```
-  Which will update the `pyproject.toml` and `pixi.lock` files with the new
-  dependency.
+
+We use pixi for development. If you would like to contribute to this repository or want to build your own template, we strongly recommend using Pixi.
+
+#### uv
+
+If you prefer using [uv](https://docs.astral.sh/uv/#tool-management), you can install `copier` with the following command:
+
+```bash
+uv tool install copier
+```
+
+Note: If you use `poetry`, you might want to consider moving to `uv`.
+
+#### pipx
+
+If you have [pipx](https://pypa.github.io/pipx/) installed, you can install `copier` with the following command:
+
+```bash
+pipx install copier
+```
+
+#### Nix flake
+
+If you are using [Nix flakes](https://nixos.wiki/wiki/Flakes), you can install `copier` with the following command:
+
+```bash
+nix profile install 'https://flakehub.com/f/copier-org/copier/*.tar.gz'
+```
+
+### 2) Build your package
+
+```bash
+copier copy https://github.com/castelao/NREL-pypackage-template ../my_new_project
+cd ../my_new_project
+```
+
+You should see several files and directories created from the information
+that you gave. For instance, take a look at the `pyproject.toml` file.
+Your package will probably depend on other libraries. Let's assume that
+you'll use `xarray`, so you can run
+```bash
+pixi add xarray
+```
```

Comment:
This depends on using pixi, right? Is it worth adding instructions if users are using homebrew or pipx or something else instead?

Reply:
Good catch. The project template is ready for pixi and assume one would use it, but conflicts with the previous instructions by allowing install with other tools. Shall we just assume that everyone should use pixi and simplify the instructions? 1) install pixi 2) create an environment to install copier 3) copy template.

Reply:
I'm of course happy with that, but would that potentially prevent more people from using it? Or are we not targeting people with an opinion around that anyways?

Reply:
I particularly think that it would be better to just assume the user will use `pixi`. This is also a lot easier to keep instructions up to date.

Reply:
A compromise here would he to have the QuickStart assuming the environment that we work with, such as `pixi` `GitHub`, .... And in the documentation expand for other options.

I'll move ahead this PR as it is because the current instructions are quite limited, and later move some stuff to a documentation.